[BUG] Simple QP example in C not compiling

**Describe the bug**

The `simple_qp_example.c` example fails to compile.
I obtain the following error message:

```c
docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c: In function 'test_simple_qp':
docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c:133:40: error: passing argument 16 of 'cuOptCreateQuadraticProblem' from incompatible pointer type [-Wincompatible-pointer-types]
  133 |                                        variable_types,
      |                                        ^~~~~~~~~~~~~~
      |                                        |
      |                                        char *
In file included from docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c:25:
~/miniconda3/envs/cuopt_dev/include/cuopt/linear_programming/cuopt_c.h:311:29: note: expected 'void **' but argument is of type 'char *'
  311 |   cuOptOptimizationProblem* problem_ptr);
      |   ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c:118:12: error: too many arguments to function 'cuOptCreateQuadraticProblem'
  118 |   status = cuOptCreateQuadraticProblem(num_constraints,
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
~/miniconda3/envs/cuopt_dev/include/cuopt/linear_programming/cuopt_c.h:295:13: note: declared here
  295 | cuopt_int_t cuOptCreateQuadraticProblem(
      |             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Note that the error occurs because the call to `cuOptCreateQuadraticProblem` should not include variable types (i.e. continuous vs integer vs binary) here:
https://github.com/NVIDIA/cuopt/blob/787243b8cedc409d9dfa7465b52d12ffd8674c08/docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c#L133

**Steps/Code to reproduce bug**

* Checkout `main` branch
* Build cuopt from source
* Compile 
    ```bash
    gcc -I . -L . -o simple_qp docs/cuopt/source/cuopt-c/lp-milp/examples/simple_qp_example.c -lcuopt
    ```

**Expected behavior**

The example should compile without error, and the executable should execute as expected.

**Environment details (please complete the following information):**
 - Environment location: DGX Spark workstation
 - Method of cuOpt install: from source using `conda` (via miniconda as specified in official instructions)


**Additional context**

Add any other context about the problem here.

--- a/docs/cuopt/source/cuopt-c/lp-qp-milp/examples/simple_qp_example.c
+++ b/docs/cuopt/source/cuopt-c/lp-qp-milp/examples/simple_qp_example.c
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-FileCopyrightText: Copyright (c) 2025-2026, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
 /*
@@ -130,7 +130,6 @@ cuopt_int_t test_simple_qp()
                                        constraint_rhs,
                                        var_lower_bounds,
                                        var_upper_bounds,
-                                       variable_types,
                                        &problem);
   if (status != CUOPT_SUCCESS) {
     printf("Error creating problem: %d\n", status);

